app/assets/javascripts/admin/views/help/keyboardshortcut_view.js

```diff
@@ -1,231 +1,257 @@
 // @flow
 import React from "react";
+import { Table } from "antd";
 
-const KeyboardShortcutView = () => (
-  <div className="container help">
-    <h3>Keyboard Shortcuts</h3>
-    <p>Find all available keyboard shortcuts for webKnossos listed below:</p>
+const KeyboardShortcutView = () => {
+  const columns = [
+    {
+      title: "Key Binding",
+      dataIndex: "keybinding",
+      key: "keybinding",
+    },
+    {
+      title: "Action",
+      dataIndex: "action",
+      key: "action",
+    },
+  ];
 
-    <h3>Orthogonal Mode</h3>
-    <table className="table table-hover table-striped">
-      <thead>
-        <tr>
-          <th>Key binding</th>
-          <th>Action</th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr>
-          <td>Left Mouse drag or Arrow keys</td>
-          <td>Move</td>
-        </tr>
-        <tr>
-          <td>I, O or Alt + Mousewheel</td>
-          <td>Zoom in/out</td>
-        </tr>
-        <tr>
-          <td>F, D or Mousewheel</td>
-          <td>Move along Z-Axis</td>
-        </tr>
-        <tr>
-          <td>Right click</td>
-          <td>Set node</td>
-        </tr>
-        <tr>
-          <td>Shift + Alt + Left click</td>
-          <td>Merge two trees</td>
-        </tr>
-        <tr>
-          <td>K, L</td>
-          <td>Scale up/down viewport size</td>
-        </tr>
-        <tr>
-          <td>B, J</td>
-          <td>Set/Jump to last branchpoint</td>
-        </tr>
-        <tr>
-          <td>S</td>
-          <td>Center active node</td>
-        </tr>
-      </tbody>
-    </table>
+  const orthogonalShortcuts = [
+    {
+      key: "1",
+      keybinding: "Left Mouse drag or Arrow keys",
+      action: "Move",
+    },
+    {
+      key: "2",
+      keybinding: "I, O or Alt + Mousewheel",
+      action: "Zoom in/out",
+    },
+    {
+      key: "3",
```

Comment:
Can we do something clever here, so it won't be such a pain to insert a new key combo in the middle (all subsequent key values will need to be increased by one in that case)?

Reply:
We could use random keys? `Math.random()`?
Otherwise I used a sublime plugin called `Increment Selection` that lets you insert number sequences with the multi cursor.

Reply:
Can we use the `keybinding` as key? `.map(a => Object.assign(a, { key: a.keybinding })`